package.json

```diff
@@ -54,6 +54,7 @@
     "start:storybook": "start-storybook -p 9009 -s public --quiet",
     "prepare": "husky install"
   },
+  "proxy": "http://localhost:5000",
```

Comment:
This allows local app to automatically configure itself through firebase emulators